test/linter.test.js

```diff
@@ -176,6 +176,31 @@ describe('linter.js', () => {
                         done();
                     });
                 });
+
+                context('when trying to figure this damn error out', () => {
+                    const rule = {
+                        name: "reference-components-regex",
+                        object: "reference",
+                        enabled: true,
+                        description: "reference components should all match regex ^[a-zA-Z0-9\\.\\-_]+",
+                        pattern: {
+                            property: "$ref",
+                            omit: "#",
+                            split: "/",
+                            value: "^[a-zA-Z0-9\\.\\-_]+$"
+                        }
+                    };
+
+                    it('should error if missing ./', done => {
```

Comment:
This test is failing, as missing a `./` is showing up as valid... this seems to be the opposite of the complaints.